docs/Documentation.md

```diff
@@ -452,7 +452,8 @@ else c.ToString()
 ### fsharp_max_infix_operator_expression
```

Comment:
Would it not be easier to understand if this setting doesn't apply for the newline infix operators?
Then you don't need to have any `*_multiline_formatter` setting in place and the default behaviour sorta is respected.